imports/ui/templates/components/decision/tag/tag.html

```diff
@@ -1,12 +1,12 @@
 <template name="tag">
-  <li class="w-clearfix tag {{authorization}} {{drag nonDraggable}}" id="draggable" value="{{../_id}}">
+  <li class="w-clearfix tag {{authorization}} {{drag nonDraggable}}" value="{{../_id}}">
     <div class="tag-label">
       <img src="{{pathFor route='home'}}images/hashtag.png" class="hash-icon">
       {{#if suggestion}}
         <a href="{{../url}}" class="w-clearfix w-inline-block hash-link {{authorization hover=true}}">
           <div class="hash-tag">{{{../text}}}</div>
         </a>
-        <div id="add-suggested-tag" class="action">
+        <div id="add-suggested-tag-{{../keyword}}" class="add-suggested-tag action">
```

Comment:
A unique id reconstructible from the Tag name is required by the step definitions, to target the button and click on it.